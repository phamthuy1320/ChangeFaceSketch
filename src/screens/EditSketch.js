import React, {useEffect} from 'react';
import {View} from 'react-native';
// import {useRoute} from '@react-navigation/native';

// import RNSketchCanvas from '@terrylinla/react-native-sketch-canvas';
import PhotoEditor from 'react-native-photo-editor'

const EditSketch = ({route}) =>{
    // const route = useRoute()
    const _onPress = () => {
        PhotoEditor.Edit({
        path: route.params?.uri.split('file://')[1],
        stickers: [
            'sticker0',
            'sticker1',
            'sticker2',
            'sticker3',
            'sticker4',
            'sticker5',
            'sticker6',
            'sticker7',
            'sticker8',
            'sticker9',
            'sticker10',
        ],
        colors:undefined,
        onDone: (imagePath) => {
            console.log('on done',JSON.stringify(imagePath));
        },
        onCancel: () => {
            console.log('on cancel',route?.params?.uri);
        },
        });
    }
    useEffect(()=>_onPress)
    return(<View/>)
}
export default EditSketch;