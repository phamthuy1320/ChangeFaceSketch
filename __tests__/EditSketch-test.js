import React from 'react';
import renderer from 'react-test-renderer';
import EditSketch from '../src/screens/EditSketch';

test('Edit Sketch Screen', () => {
  const tree = renderer.create(<EditSketch/>).toJSON();
  expect(tree).toMatchSnapshot();
});