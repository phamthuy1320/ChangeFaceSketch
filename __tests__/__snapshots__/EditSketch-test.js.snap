// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Sketch Screen 1`] = `<View />`;
